src/app/data/components/data-browser/data-browser.component.html

```diff
@@ -1288,4 +1004,241 @@ <h1 class="text-lg text-gray-900 mb-2">Info</h1>
+                        <input #inputDynamic type="radio" style="--focus-shadow:none;accent-color:rgb(29,78,216)"
+                            (click)="setStatic(inputDynamic,inputStatic,false)"
+                            class="h-4 w-4 border-gray-200 cursor-pointer" checked [value]="true">
+                    </label>
+                </div>
+                <div class="form-control">
+                    <label class="cursor-pointer label">
+                        <span class="label-text mr-1">Static</span>
+                        <input #inputStatic type="radio" style="--focus-shadow:none;accent-color:rgb(180,83,9)"
+                            (click)="setStatic(inputDynamic,inputStatic,true)"
+                            class="h-4 w-4 border-gray-200 cursor-pointer" [value]="false">
+                    </label>
+                </div>
+            </div>
+            <div data-tip="Too many records for a static slice" class="relative tooltip mr-1"
+                style="white-space: break-spaces;">
+                <div class="flex justify-center items-center w-full h-full"
+                    [ngClass]="{hidden:!dataBrowserModals.filter.displayStaticNotAllowedWarning}">
+                    <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24"
+                        stroke="currentColor" stroke-width="2">
+                        <path fill="#E8CD20" stroke-linecap="round" stroke-linejoin="round"
+                            d="M12 9v2m0 4h.01m-6.938 4h13.856c1.54 0 2.502-1.667 1.732-3L13.732 4c-.77-1.333-2.694-1.333-3.464 0L3.34 16c-.77 1.333.192 3 1.732 3z" />
+                    </svg>
+                </div>
+            </div>
+        </div>
+        <div class="flex w-full justify-between mt-2">
+            <input #inputSliceName type="text" class="w-1/2 input input-bordered input-sm" placeholder="Enter name"
+                name="dataSliceName" [value]="dataBrowserModals.filter.name"
+                (input)="setFilterName(inputSliceName.value)" (keyup)="checkNameExists(inputSliceName.value)"
+                (keydown.enter)="dataBrowserModals.filter.open = false; dataBrowserModals.filter.sliceNameExists ?
+            updateSliceByName():createSlice()">
+        </div>
+    </div>
+
+</kern-modal>
+
+<kern-modal [isOpen]="dataBrowserModals.sliceInfo.open" closeButton="X"
+    (optionClicked)="dataBrowserModals.sliceInfo.open = false">
+    <div class="flex flex-grow justify-center mb-4 font-bold">Slice information</div>
+    <div *ngFor="let item of dataBrowserModals.sliceInfo.data | keyvalue : orderOriginal">
+        <ng-template [ngIf]="item.key == 'Link'" [ngIfElse]="normalDisplay">
+            <div class="mt-3 flex rounded-md">
+                <span
+                    class="inline-flex items-center rounded-l-md border border-r-0 border-gray-300 bg-gray-50 px-3 text-gray-500 sm:text-sm">{{item.value.startsWith("https")?
```

Comment:
function call inside html -> i think it was there before the change but since I am noticing some performance issues I think we need to remove as many as possible :)